osu.Game.Tests/Visual/TestMultiplayerComponents.cs

```diff
@@ -73,7 +73,8 @@ public TestMultiplayerComponents()
         [BackgroundDependencyLoader]
         private void load(IAPIProvider api)
         {
-            ((DummyAPIAccess)api).HandleRequest = request => requestsHandler.HandleRequest(request, api.LocalUser.Value, beatmapManager);
+            if (api is DummyAPIAccess dummy)
+                dummy.HandleRequest = request => requestsHandler.HandleRequest(request, api.LocalUser.Value, beatmapManager);
```

Comment:
In the case of the multiplayer navigation test scenes, this is now a `OsuGameTestScene.DelegatedAPI`.